app/konnect/runtime-manager/vaults/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Secrets Management in Konnect
+content_type: explanation
+---
+
+Secure your {{site.konnect_short_name}} environment with centralized secrets,
+making it easier to manage and build governance.
+
+A secret is a sensitive piece of information required for proper gateway operations.
+Secrets may be part of the core gateway configuration,
+or part of gateway configuration associated with APIs serviced by the gateway.
+Some of the most common types of secrets include:
+* Certificates
+* API keys
+* Personal access tokens
+* Credentials for databases
+* Certain plugin fields, like `session_secret` in the OIDC plugin
+
+You can use vaults to safely store and retrieve secrets used in {{site.base_gateway}}
+deployments, improving the fundamental security of your applications.
+In configuration, you can then reference the secrets stored in vaults instead
+of displaying the actual value of the secret.
+
+This ensures that secrets are not stored in plaintext throughout the platform,
+in places such as `kong.conf` files, in declarative configuration files, logs,
+or in the {{site.konnect_short_name}} UI. This way, the {{site.konnect_short_name}}
+platform never stores or sees sensitive credentials, letting you use the security controls
+that you already have around sensitive information.
+
+## Vaults interface in {{site.konnect_short_name}}
+
+![Vaults interface](/assets/images/docs/konnect/konnect-vaults.png)
+> Figure 1: Overview page for all vaults configured for a runtime group.
+
+Number | Item | Description
+-------|------|------------
+1 | **Vaults menu link** | Main link to the vaults configuration for a runtime group. Appears when you select a runtime group.
+2 | **Add vault** | Click the **Add vault** button to set up any supported Konnect vault backend.
+3 | **Table entries for vaults** | Select a table entry to see the configuration for a particular vault. On each vault's configuration page, you can edit or delete the vault, or copy the entire configuration as JSON.
+4 | **Vault ID** | The vault's UUID. You can click the copy icon to copy the ID for use in configuration.
```

Comment:
```suggestion
4 | **Vault ID** | The vault's UUID. 
```

Reply:
Hmm. I'd like to leave that out for now. I'm not 100% sure why you'd need to do that at the moment, because there's no way to edit it through the API (I think?). That would be the main reason to use a vault UUID.